addons/io_scene_gltf2/blender/imp/gltf2_blender_map_occlusion.py

```diff
@@ -32,12 +33,75 @@ def create_nodetree(gltf, material_idx, vertex_color):
         """Nodetree creation."""
         pymaterial = gltf.data.materials[material_idx]
 
+        material = bpy.data.materials[pymaterial.blender_material[vertex_color]]
+        node_tree = material.node_tree
+
         BlenderTextureInfo.create(gltf, pymaterial.occlusion_texture.index)
 
-        # Pack texture, but doesn't use it for now. Occlusion is calculated from Cycles.
+        # Pack texture. Occlusion is calculated from Cycles or Eevee, so do nothing
```

Comment:
Minor, but I didn't think you could bake occlusion in Eevee yet?

Reply:
Oh, or this means that rendering in Blender will calculate AO one way or the other?

Reply:
Yes, I mean that AO is not provided by user, but the rendering engine process leads to an AO calculation. (And you're right, we can't bake it)